doc/frontend-notes.txt

```diff
@@ -0,0 +1,70 @@
+Basically one can run any HTTP-Proxy infront (frontend) of the ruby rack
+application server (like puma or webrick), which in turn provides access to
+the ontohub web application. These notes provides some hints wrt. the frontends
+we use or collected from other contributors.
+
+Apache HTTPd configuration
+--------------------------
+Setup your apache httpd as usual. If in doubts or you don't reall know, what
+a directive mentioned below does or how to activate a certain httpd module,
+please consult your OS distro vendor's manual.
+
+Set the document root to ontohub's "public" directory (e.g.
+"DocumentRoot /local/home/ontohub/ontohub/public").
+
+Make sure, that the rewrite module is activated, enable the rewrite engine
+("RewriteEngine On"), and put the follwing two blocks before any proxy
+directives:
+
+	# Maintenance mode
+	RewriteCond /data/git/maintenance.txt -f
```

Comment:
I mean this should be the directory specified in `settings.yml` at `paths.data` (after the last settings pull request is merged)